iroh-relay/src/client/conn.rs

```diff
@@ -443,7 +432,7 @@ impl TryFrom<Frame> for ReceivedMessage {
             Frame::Pong { data } => Ok(ReceivedMessage::Pong(data)),
             Frame::Health { problem } => {
                 let problem = std::str::from_utf8(&problem)
-                    .map_err(|_| RecvError::InvalidProtocolMessageEncoding)?
+                    .context(InvalidProtocolMessageEncodingSnafu {})?
```

Comment:
I think you can leave out the curly braces

Reply:
Yeah you're right. Not sure why I thought I couldn't... fixed